### PR TITLE
0008-migrate-create-toast-component

### PR DESCRIPTION
**Pull request by giselalanciani**
CR to check

### CODE REVIEW

File: dist/categories/create/index.html [hunk 1]
```diff
@@ -17,16 +17,41 @@ <h1>Create categories</h1>
...
+          class="btn btn-primary"
+        >
+          Create
+        </button>
       </form>
-    </main>        
+    </main>
+    <div class="position-fixed bottom-0 end-0 p-3" style="z-index: 11">
+      <div
+        id="delete-toast"
```
Comment (diegolaciar) [inline dist/categories/create/index.html]: Change all the ID `id="delete-toast"`. in this case to id="create-toast"
You will. have to change also the ID on the TS. files.

File: src/controllers/authors/create-authors-controller.ts [hunk 2]
```diff
@@ -128,14 +126,21 @@ class CreateAuthorsController {
       name: authorNameInputElement.value,
       birthdate: birthdate.toISOString(),
     };
-
+    const toastModalElement = <HTMLDivElement>(
+      document.querySelector("#delete-toast")
```
Comment (diegolaciar) [inline src/controllers/authors/create-authors-controller.ts]: This ID. on all the controllers should be.   create-toast

**Review by diegolaciar — CHANGES_REQUESTED** [review]
@giselalanciani,

Just check the comments and fix them. 
Test the changes.

Then it will be ready for merge.

File: dist/categories/create/index.html (hunk 1 above)
Comment (giselalanciani) [reply dist/categories/create/index.html]: Work Done

File: src/controllers/authors/create-authors-controller.ts (hunk 2 above)
Comment (giselalanciani) [reply src/controllers/authors/create-authors-controller.ts]: work done

**Comment by giselalanciani** [comment]
all changes ready 
@diegolaciar

**Review by diegolaciar — APPROVED** [review]
@giselalanciani ,

Well done !

Ship it! 🚢

**Pull request merged**